db/seeds.rb

```diff
@@ -5,3 +5,20 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+require 'faker'
+
+
+5.times do
+  User.create(name: Faker::Name.name, photo:'https://unsplash.com/photos/F_-0BxGuVvo', bio: Faker::Lorem.sentence)
+end
+
+30.times do
+  Post.create(user_id: Faker::Number.between(from: 1, to: 5), title: Faker::Lorem.word, text: Faker::Lorem.paragraph_by_chars)
+end
+
+200.times do
+  Comment.create(post_id: Faker::Number.between(from: 1, to: 30), user_id: Faker::Number.between(from: 1, to: 5),text:Faker::Lorem.sentence)
+end
+200.times do
+  Like.create(post_id: Faker::Number.between(from: 1, to: 30), user_id: Faker::Number.between(from: 1, to: 5))
```

Comment:
- [ ] Also please remove these lines as they are not permitted in the seed the data should be added via the` Rails c`, they should be pushed not added.

Reply:
Done